inst/sass-color/app.R

```diff
@@ -0,0 +1,65 @@
+library(shiny)
+library(sass)
+library(colourpicker)
+
+input_scss <- "new-style.scss"
+output_css <- "new-style.css"
+
+style_list <- list(
+  `$color` = "#6498d2"
+)
+
+sass(input_scss, output = output_css)
+
+ui <- fluidPage(
+  headerPanel("Sass Color Example"),
+
+  sidebarPanel(
+    colourInput("color", "Background Color", value = style_list[[1]],
+                showColour = "text")
+  ),
+
+  mainPanel(
+    plotOutput("distPlot"),
+    br()
+  ),
+
+  column(6, verbatimTextOutput("scss")),
+  column(6, verbatimTextOutput("css")),
+
+  uiOutput("bgcolor")
+)
+
+server <- function(input, output) {
+  output$distPlot <- renderPlot({
+    hist(rnorm(500))
+  })
+
+  variables <- reactive({
+    style_list$`$color` <- input$color
+    style_list$`$width` <- input$width
+    paste0(names(style_list), ": ", style_list, ";", collapse = '\n')
+  })
+
+  compiled_css <- reactive({
+    write(variables(), "_variables.scss")
+    sass(input_scss, output = output_css)
+  })
+
+  output$bgcolor <- renderUI({
+    compiled_css()
+    includeCSS(output_css)
```

Comment:
Don't use `includeCSS(css_file)`, instead do `tags$style(HTML(css_text))` (or have `sass()` return an object that's tagged as HTML).